sessionctx/variable/sysvar.go

```diff
@@ -1325,6 +1325,15 @@ var defaultSysVars = []*SysVar{
 		s.EnablePaging = TiDBOptOn(val)
 		return nil
 	}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBReadConsistency, Value: Off, Type: TypeStr, Hidden: true, skipInit: true, SetSession: func(s *SessionVars, val string) error {
```

Comment:
I have two questions here:
- when `skipInit` is true, [the session cache](https://github.com/pingcap/tidb/blob/722bd9ff9d24ddaa493969ab91ec2d00a6556a1b/session/session.go#L2860) won't contains this variable, thus `set global ...` won't take effect. Do you intend to do so?
- if this variable is set to weak globally, will it affect internal sql?

Reply:
The comments is ambiguous. Seems it indeed won't take effect in new session, but many sysvar with global scope set it as true.
```go
	// skipInit defines if the sysvar should be loaded into the session on init.
	// This is only important to set for sysvars that include session scope,
	// since global scoped sysvars are not-applicable.
	skipInit bool
```

Reply:
This sysvar is very dangerous. We must guarantee only non-internal autocommit readonly query can use it.

Reply:
Good catch for `skipInit`.